### ISSUE TITLE
Cant equip helmet on NPC who has salvaged power armor on. 

### ISSUE THREAD

**Issue by Orange01gaming** [CleverRaven/Cataclysm-DDA#26321]
**Describe the bug**  
I am can't equip headgear or much else for that matter to the NPC. Also they appear naked. 

**To Reproduce**  
1. Get salvaged power armor, a helmet and NPC
2. Press e on them, then go to sort armor, and press e to equip armor. After putting on salvaged power armor you can't put on a helmet. 

**Expected behavior**  
You should be able to get power armor helmets on, and not be able to layer on other gear over/under the salvaged power armor

**Screenshots**  

**Versions and configuration(please complete the following information):**
 - OS: [Windows 10]
 - Game Version [from the main menu, e.g. 0.C-33357-g1f2a55c]
Graphics version Tiles
Mods loaded [Filthy Clothes, NPC traits, More Buildings, PK rebalanced]

**Comment by OliveChukar** [comment]
Tested in ver 0.E-9072-g1bb53ba-dirty with core content pack dark days ahead only.

The main issue in this report seems to be fixed.

 I was able to get the NPC to equip salvaged power armor and a salvaged power armor helmet. I was also able to get the NPC to equip a dragonskin vest a thermal electric outfit and a zentai at the same time with the salvaged power armor. I do not know if the ability to equip clothing with power armor is intended so I am not going to report it.
